types/cytoscape/index.d.ts

```diff
@@ -739,7 +744,7 @@ declare namespace cytoscape {
          *
          * @param bool A truthy value enables panning; a falsey value disables it.
          */
-        panningEnabled(bool: boolean): this;
+        panningEnabled(bool?: boolean): this;
```

Comment:
Please confirm that you wanted to allow `panningEnabled()`, in addition to `panningEnabled(undefined)`.

Reply:
You're right, `panningEnabled(bool: boolean | undefined): this;` would be more suitable. However, there is already `panningEnabled(): boolean;`, which takes precedence. I followed the pattern existing with other similar methods, to limit the amount of changes. Shall I fix it in this PR?

Reply:
No, you're good.  GH was hiding the overload.  Thanks for clarifying!